### PR TITLE
Doc review

### PR DESCRIPTION
**Pull request by aadcg**

### CODE REVIEW

File: source/prompt-buffer.lisp [hunk 1]
```diff
@@ -35,11 +35,11 @@ new history for each new prompt buffer.  Here we set the history to be shared gl
      ;; TODO: Need a changed-callback?  Probably not, see `search-buffer'.  But
      ;; can we run the postprocessor without running the filter?
      (invisible-input-p nil
-                        :documentation "If non-nil, input is replaced by placeholder character.
-;; This is useful to conceal passwords.")
+                        :documentation "If non-nil, input is replaced by
```
Comment (Ambrevar) [inline source/prompt-buffer.lisp:38]: While we are at it, should be "a placeholder character", no?

**Review by Ambrevar — COMMENTED** [review]
All good catches, thank you!

**Comment by aadcg** [comment]
Should be good now!

**Comment by Ambrevar** [comment]
Thanks!

**Pull request merged**